dips/dip-10.md

```diff
@@ -13,36 +13,36 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # Summary
 
-This DIP describes Diem ID - the human-readable identifier for user accounts.   
+This DIP describes DiemID - the human-readable identifier for user accounts.   
 
 # Motivation
-Diem ID provides a convenient method for identifying users within a VASP. Diem ID allows users to exchange human-readable identifiers as either the sender or the recipient of peer-to-peer payments, and plays the role of an email address for payments. The benefits of using a Diem ID are:
-* Privacy: Diem ID needs to be used in tandem with an off-chain protocol between VASPs in order exchange information about the end user corresponding to the Diem ID. Hence, Diem ID eliminates the need to directly include user information in a transaction and expose personally identifiable information (PII) on the chain 
-* Persistent Identifiers: Currently there is a lack of persistent user identifiers in the Diem ecosystem. Diem ID is a persistent identifier from a user's perspective. From the perspective of the chain, Diem IDs do not exist.
+DiemID provides a convenient method for identifying users within a VASP. DiemID allows users to exchange human-readable identifiers as either the sender or the receiver of peer-to-peer payments, and plays the role of an email address for payments. The benefits of using a DiemID are:
+* Privacy: DiemID needs to be used in tandem with an off-chain protocol between VASPs in order to exchange information about the end user corresponding to the DiemID. By initializing an off-chain preflight to exchange a reference ID, we do not need to include any potentially identifiable user information in a transaction 
+* Persistent Identifiers: Currently there is a lack of persistent user identifiers in the Diem ecosystem. DiemID is a persistent identifier from a user's perspective. From the perspective of the chain, DiemIDs do not exist
 
 # End-to-End Experience
-Below is an example of using Diem ID domain for transferring money from one user to another. 
+Below is an example of using DiemID domain for transferring money from one user to another. 
 
 ## Prerequisite:
-* VASPs get approval from association (via some offline process) on domain name
+* VASPs get approval from association (via some offline process) on domain name. VASPs optionally have locally stored a mapping of domain ID to parent VASP address by reading the DiemID event stream under Treasury Compliance on-chain account:
```

Comment:
Maybe make two bullets here

```suggestion
* VASPs get approval from association (via some offline process) on domain name.
* VASPs have access to a storage solution mapping  domain IDs to parent VASP address
```

the rest are implementation details discussed below